02week/bio/css/style.css

```diff
@@ -0,0 +1,90 @@
+body {
+    background-image: url(../edd.jpg);
```

Comment:
it's conventional to create an img/image folder to store all of the images you would use for the project.

Reply:
Okay, I will definitely take that approach, moving forward! Thank you!